source/js/components/currency-selector.js

```diff
@@ -140,9 +134,6 @@ class CurrencySelect {
     this.selectMenu.addEventListener("change", () => {
       this.getSelectValue();
     });
-
-    // Initial defaults
-    this.processSelectDefaultValue();
```

Comment:
Have added this back in, we need to check if the default value has any payment methods disabled, so I've moved this into this call.